src/material/table/_table-flex-styles.scss

```diff
@@ -0,0 +1,71 @@
+/**
+ * Flex-based table structure
+ */
+$mat-header-row-height: 56px;
+$mat-row-height: 48px;
+$mat-row-horizontal-padding: 24px;
+
+@mixin _mat-table-flex-styles {
```

Comment:
can you add a comment to only use tag name selectors here, not class selectors, since the styles are shared between MDC and non-MDC